src/run.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Internal dependencies
+ */
+const { formatReport } = require( './utils/format-report' );
+const { getArg } = require( './utils/cli' );
+
+const runAudits = ( cssFiles ) => {
+	const audits = [];
+	const runAll = getArg( '--all' );
+	const runRecommended = getArg( '--recommended' );
+
+	if ( runAll || runRecommended || getArg( '--colors' ) ) {
+		audits.push( require( './audits/colors' )( cssFiles ) );
+	}
+	if ( runAll || runRecommended || getArg( '--important' ) ) {
+		audits.push( require( './audits/important' )( cssFiles ) );
+	}
+	if ( runAll || getArg( '--display-none' ) ) {
+		audits.push( require( './audits/display-none' )( cssFiles ) );
+	}
+	if ( runAll || runRecommended || getArg( '--selectors' ) ) {
+		audits.push( require( './audits/selectors' )( cssFiles ) );
+	}
+	if ( runAll || runRecommended || getArg( '--media-queries' ) ) {
+		audits.push( require( './audits/media-queries' )( cssFiles ) );
+	}
+
+	const propertyValues = getArg( '--property-values' );
+
+	// Multiple property value arguments are only supported in config.
+	if ( Array.isArray( propertyValues ) && propertyValues.length ) {
+		propertyValues.forEach( ( values ) => {
+			audits.push(
+				require( './audits/property-values' )(
+					cssFiles,
+					values.split( ',' )
+				)
+			);
+		} );
+	} else {
+		audits.push(
+			require( './audits/property-values' )(
+				cssFiles,
+				propertyValues.split( ',' )
```

Comment:
I changed the css-audit.config.js file to 

```
module.exports = {
	format: 'html',
	filename: 'wp-admin',
	audits: [ 'colors' ],
};
```

and it triggers an error: `TypeError: propertyValues.split is not a function`